### PR TITLE
add triplets_with_relations

### PR DESCRIPTION
**Pull request by itayzit**

### CODE REVIEW

File: knowledge_graph/querying_raw.py [hunk 1]
```diff
@@ -70,9 +70,19 @@ def triplets_with_relation(
...
-all_display_and_trigger_triplets = gamla.juxtcat(
-    triplets_with_relation(common_relations.TRIGGER),
-    triplets_with_relation(common_relations.DISPLAY),
+def triplets_with_relations(
+    relations: Iterable[triplet.Element],
+) -> Callable[[triplets_index.TripletsWithIndex], Iterable[triplet.Triplet]]:
+    return gamla.pipe(
+        relations,
+        gamla.map(triplets_with_relation),
+        gamla.star(gamla.juxt),
```
Comment (uriva) [inline knowledge_graph/querying_raw.py]:
```suggestion
        gamla.star(gamla.juxctat),
```
(and remove the line below)

File: knowledge_graph/querying_raw.py [hunk 2]
```diff
@@ -70,9 +70,19 @@ def triplets_with_relation(
     return gamla.compose_left(triplets_index.relation_index, gamla.apply(relation))
 
 
-all_display_and_trigger_triplets = gamla.juxtcat(
-    triplets_with_relation(common_relations.TRIGGER),
-    triplets_with_relation(common_relations.DISPLAY),
+def triplets_with_relations(
```
Comment (uriva) [inline knowledge_graph/querying_raw.py]: point free

File: knowledge_graph/common_relations.py [hunk 3]
```diff
@@ -30,7 +30,6 @@
 EVENT_DATA = "event/data"
 EVENT_TYPE = "event/type"
 ACTION_TEXT_VALIDATION = "action/text_validation"
-CONCEPT_TRIGGER = "concept/trigger"
```
Comment (uriva) [inline knowledge_graph/common_relations.py:33]: on purpose?

**Review by uriva — APPROVED** [review]

File: knowledge_graph/common_relations.py (hunk 3 above)
Comment (itayzit) [reply knowledge_graph/common_relations.py:33]: yes, it already exists under a different name.

**Pull request merged**